doc/ref_model/chapters/chapter03.md

```diff
@@ -298,6 +298,16 @@ It is expected that reference architectures will provide a service chain workflo
 
 There is also a need to provide specialised tools to aid troubleshooting of individual services and the communication between them in order to investigate issues in the performance of composed network functions. Minimally, there is a need to provide packet level and byte level counters and statistics as the packets pass through the service chain in order to ascertain any issues with forwarding and performance. Additionally, there is a need for mechanisms to trace the paths of selected subsets of traffic as they flow through the service chain.
 
+<a name="3.5.1.1"></a>
+#### 3.5.1.1 Introduction 
```

Comment:
This is more or less redundant to the text in chapter 3.5.1. Can you please merge these two?

Reply:
@CsatariGergely removed the line mentioning acyclic graph as it is already part of 3.5.1

Reply:
I would still merge the two chapters as both of them are some kind of introduction.

Reply:
@CsatariGergely  i feel introduction is bad title for sub section .. can we rename it as "SFC layering model " rather than introduction ? will it make it more specific or still we merge it with previous section?